src/main/java/io/vertx/kafka/client/consumer/KafkaConsumer.java

```diff
@@ -176,6 +176,18 @@
   @Override
   KafkaConsumer<K, V> endHandler(Handler<Void> endHandler);
 
+  /**
+   * Returns the current fetch amount.
+   *
+   * <ul>
+   *   <i>If the stream is in <i>flowing</i> mode will return {@link Long#MAX_VALUE}.</i>
+   *   <li>If the stream is in <i>fetch</i> mode, will return the current number of elements still to be delivered or 0 if paused.</li>
+   * </ul>
+   *
+   * @return current fetch amount
+   */
+  long currentFetchAmount();
```

Comment:
This should be named `demand()`, the fetch is the value added to the current demand to produce a new demand value

Reply:
done